src/common/combocmn.cpp

```diff
@@ -2417,6 +2421,15 @@ void wxComboCtrlBase::SetButtonBitmaps( const wxBitmap& bmpNormal,
     RecalcAndRefresh();
 }
 
+wxBitmap wxComboCtrlBase::GetBitmapFromBundle(const wxBitmapBundle& bundle) const
```

Comment:
We probably should add this as a function somewhere (`wxBitmapBundle::GetBitmapFor()` static overload?), rather than having it in this class -- this is really not specific to it at all.